app/javascript/packs/multi_serial_search/api.js

```diff
@@ -0,0 +1,40 @@
+/*
+  Private
+*/
+
+const url = urn => (process.env.RAILS_ENV === 'development'
+  ? `http://localhost:3001/${urn}`
+  : `https://bikeindex.org/${urn}`);
+
+const serialSearchUrl = serial => url(
+  `api/v1/bikes?multi_serial_search=true&serial=${serial}`,
```

Comment:
Can you switch to v2 or v3 here?

Reply:
Sorry for not making it clear that I wanted this!

Reply:
No problem!  I'll swap it in